app/src/lib/editors/found-editor.ts

```diff
@@ -2,4 +2,5 @@ export interface IFoundEditor<T> {
   readonly editor: T
   readonly path: string
   readonly usesShell?: boolean
+  useWorkspace?: boolean
```

Comment:
Unless this needs to be mutable, we prefer to make things `readonly` by default

```suggestion
  readonly useWorkspace?: boolean
```